README.md

```diff
@@ -4,798 +4,3963 @@
-     ```
-      $ sudo ./copy_certificate.sh
-     ```
-     The following files are copied in the path - `/etc/odimracert/`:
-      - rootCA.crt
-      - odimra_server.key
-      - odimra_server.crt
-      - odimra_rsa.public
-      - odimra_rsa.private
-      - odimra_kafka_client.key
-      - odimra_kafka_client.crt
-
-     The following files are copied in the path - `/etc/kafka/conf/`:
-      - kafka.keystore.jks
-      - kafka.truststore.jks
-
-     The following files are copied in the path - `/etc/zookeeper/conf`:
-      - zookeeper.keystore.jks
-      - zookeeper.trustore.jks
-      
-    The following files are copied in the path - `/etc/plugincert/`:
-      - rootCA.crt
-      - odimra_server.key
-      - odimra_server.crt
-      - odimra_kafka_client.key
-      - odimra_kafka_client.crt
-      
+    Skip this step if you have chosen a one-node cluster configuration.
+	
+	
+	
+## Pulling the Docker images of all the Kubernetes microservices
 
-8. Navigate to the ODIMRA folder.
-   ```
-   $ cd ~/ODIM
-   ```
+1. Run the following command to pull each Kubernetes image on the deployment node:
 
-9. Use the following command to deploy and start the containers:
    ```
-   $ make all
+   $ docker pull <imagename>:<version>
    ```
-    The following containers are loaded:
-      - build_odimra_1
-      - build_kafka_1
-      - build_zookeeper_1
-      - build_redis_1
-      - build_consul_1
-      - build_grf_plugin_1
-	  - build_urp_1
-
-10. Verify that the resource aggregator services are running successfully.
+   
+   Example: `$ docker pull calico/cni:v3.15.1`
+   
+   The following table lists all the Kubernetes images.
+   
+   |Docker image name|Version|Docker image tar file name|
+   |-----|----|-----|
+   |calico/cni|v3.15.1 |calico_cni.tar |
+   |calico/kube-controllers|v3.15.1 |calico_kube-controllers.tar |
+   |calico/node|v3.15.1 |calico_node.tar |
+   |coredns/coredns|1.6.7 |coredns_coredns.tar |
+   |k8s.gcr.io/cluster-proportional-autoscaler-amd64|1.8.1 |k8s.gcr.io_cluster-proportional-autoscaler-amd64.tar |
+   |k8s.gcr.io/k8s-dns-node-cache|1.15.13 |k8s.gcr.io_k8s-dns-node-cache.tar |
+   |k8s.gcr.io/kube-apiserver|v1.18.5 |k8s.gcr.io_kube-apiserver.tar |
+   |k8s.gcr.io/kube-controller-manager|v1.18.5 |k8s.gcr.io_kube-controller-manager.tar |
+   |k8s.gcr.io/kube-proxy|v1.18.5 |k8s.gcr.io_kube-proxy.tar |
+   |k8s.gcr.io/kube-scheduler|v1.18.5 |k8s.gcr.io_kube-scheduler.tar |
+   |k8s.gcr.io/pause|3.2 |k8s.gcr.io_pause.tar |
+   |lachlanevenson/k8s-helm|v3.2.3 |lachlanevenson_k8s-helm.tar |
+   |nginx|1.19 |nginx.tar |
+   |quay.io/coreos/etcd|v3.4.3 |quay.io_coreos_etcd.tar |
+   
+2. Verify that the images are successfully pulled using the following command:
    ```
-   $ ps -eaf | grep svc
+   $ docker images
    ```
+3. Save each Docker image to a tar archive using the following command:
+    ```
+    $ sudo docker save -o <image_name.tar> <image_name:tag>
```

Comment:
A note to refer the table above for the mapping of tar archive name for each image should be helpful